Sources/NIO/EventLoop.swift

```diff
@@ -770,6 +770,14 @@ extension EventLoopGroup {
     }
 }
 
+/// Specifies how `EventLoopGroup` will be created and establishes lifecycle ownership.
```

Comment:
Some docs on the purpose of this type could be good, e.g.

```suggestion
/// Specifies how `EventLoopGroup` will be created and establishes lifecycle ownership.
///
/// This type is intended to be used by libraries which use NIO, and offer their users either the option
/// to `.share` an existing event loop group or create (and manage) a new one (`.createNew`) and let it be 
/// managed by given library and its lifecycle.
```